openresty-build-tools/kong-ngx-build

```diff
@@ -424,16 +425,31 @@ main() {
     pushd $OPENRESTY_DOWNLOAD
       if [ ! -f Makefile ]; then
         OPENRESTY_RPATH=${OPENRESTY_RPATH:-$OPENSSL_INSTALL/lib}
-        OPENRESTY_OPTS=(
-          "--prefix=$OPENRESTY_PREFIX"
-          "--with-cc-opt='-I$OPENSSL_INSTALL/include'"
-          "--with-pcre-jit"
-          "--with-http_ssl_module"
```

Comment:
Don't use "--with-pcre-jit" because pcre doesn't build on s390x and kong.next was refined to work without pcre.